src/main/webapp/app/exercises/shared/exercise/exercise-utils.ts

```diff
@@ -170,7 +170,9 @@ export const isStartExerciseAvailable = (exercise: ProgrammingExercise): boolean
     return (
         exercise.dueDate == undefined ||
         moment() <= exercise.dueDate! ||
-        (exercise.buildAndTestStudentSubmissionsAfterDueDate == undefined && exercise.assessmentType === AssessmentType.AUTOMATIC)
+        (exercise.buildAndTestStudentSubmissionsAfterDueDate == undefined &&
+            exercise.assessmentType === AssessmentType.AUTOMATIC &&
+            !exercise.allowComplaintsForAutomaticAssessments)
```

Comment:
I prefer the `&&` to be starting the new line, but a matter of taste I guess
```suggestion
        (exercise.buildAndTestStudentSubmissionsAfterDueDate == undefined 
             && exercise.assessmentType === AssessmentType.AUTOMATIC
             && !exercise.allowComplaintsForAutomaticAssessments)
```